pyth-sdk-solana/Cargo.toml

```diff
@@ -11,7 +11,7 @@ keywords = [ "pyth", "solana", "oracle" ]
 readme = "README.md"
 
 [dependencies]
-solana-program = "1.8.1"
+solana-program = "1.8.1, < 1.11"
```

Comment:
suggest leaving a comment here explaining why < 1.11 that includes a link to the issue you created.